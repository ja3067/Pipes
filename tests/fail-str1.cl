x = "hello"

if False:
    x = 1.0

print(x[0])
x[0] = "g"
