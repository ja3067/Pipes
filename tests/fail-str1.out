h
RuntimeError: unsupported operand type(s) for list assignment
